Tasks/Maven/maventask.ts

```diff
@@ -181,7 +181,7 @@ async function execBuild() {
                             if ((options[i] === '--settings' || options[i] === '-s') && (i + 1) < options.length) {
                                 i++; // increment to the file name
                                 let suppliedSettingsXml: string = options[i];
-                                tl.cp(path.join(tl.cwd(), suppliedSettingsXml), settingsXmlFile, '');
```

Comment:
Can you explain how resolve is better than join here? Is it because suppliedSettingsXml might be a full path?

Reply:
You can just use `path.resolve(suppliedSettingsXml)`.  `resolve` will add the current working directory to create an absolute path if `suppliedSettingsXml` is relative, or return `suppliedSettingsXml` if it is absolute.

Reply:
tl.cwd() is not necessarily the current working directory.

Reply:
@jpricketMSFT: Yes, join doesn't work if the suppliedSettingsXml is a full path that is not under the cwd (see linked issue)
@brcrista : I think the way it is written is more readable :)